src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/Views/DynamicTypeTableViewCell.swift

```diff
@@ -88,4 +95,10 @@ class DynamicTypeTableViewCell: UITableViewCell, DynamicTableViewTextCell {
 		contentView.layoutMargins = UIEdgeInsets(top: 8, left: 16, bottom: 8, right: 16)
 		contentView.insetsLayoutMarginsFromSafeArea = false
 	}
+	
+	@objc
+	private func labelTapped() {
+		contentTextLabel.adjustsFontSizeToFitWidth = true
```

Comment:
That property should alway be `true` on `UILabel`s, when we want to use Dynamic Type. Independent from a gesture handler.

Reply:
Yes, I am also assuming the same but unfortunately without this property it doesn't work.